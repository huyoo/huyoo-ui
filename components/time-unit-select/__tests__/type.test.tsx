/**
 * @DECS:
 * @AUTH: hy
 * @DATE: 2021-03-29
 */
import React from "react";
import TimeUnitSelect from "../index";

describe("TimeUnitSelect.type", () => {
  it('should return hour component', function () {
    expect(2 + 2).toBe(4)
  });
})

describe("TimeUnitSelect.dom", () => {
  it('should be a component', function () {
    const dom = (
      <TimeUnitSelect type="default"/>
    );

    expect(dom).toMatchSnapshot()
  });

})
