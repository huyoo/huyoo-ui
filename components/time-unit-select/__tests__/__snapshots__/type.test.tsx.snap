// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TimeUnitSelect.dom should be a component 1`] = `
<TimeUnitSelect
  type="default"
/>
`;
